### PR TITLE
refactor: internal folder structure change

### PR DESCRIPTION
**Pull request by priyajeet**
BREAKING: changes internal file and folder structure

FolderName (pascal case) -> folder-name (kebab case)
src/components -> src/elements
src/wrappers -> src/elements/wrappers
src/components/&ast;.js -> src/elements/common/&ast;.js
src/components/&ast;.scss -> src/elements/common/&ast;.scss
lib -> es
src/util -> src/utils

screen shot 2019-01-11 at 12 28 36 am

screen shot 2019-01-11 at 12 29 19 am

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @priyajeet has signed the CLA. Thanks for the pull request!

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@
 .idea
 _site
 dist
+es
 i18n/*.js
 i18n/json
 lib
```
Comment (jstoffan) [inline .gitignore]: Remove?

File: .npmignore [hunk 2]
```diff
@@ -29,6 +29,7 @@ CODEOWNERS
 commitlint.config.js
 conf
 coverage
+cypress.json
```
Comment (jstoffan) [inline .npmignore]: Whoops!

File: src/api/Feed.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ import VersionsAPI from './Versions';
 import TasksAPI from './Tasks';
 import TaskAssignmentsAPI from './TaskAssignments';
 import { getBadItemError, getBadUserError, isUserCorrectableError } from '../util/error';
-import messages from '../components/messages';
+import messages from '../elements/common/messages';
```
Comment (jstoffan) [inline src/api/Feed.js]: It would be great to alias both `messages` and `constants` globally at some point.

Comment (priyajeet) [reply src/api/Feed.js]: Yeah

**Pull request merged**